src/daria/image/image.py

```diff
@@ -193,26 +189,33 @@ def write(
             name (str): name of image
             path (str): tpath to image (only folders)
             file_format (str): file ending, deifning the image format
-            save_metadata (bool): controlling whether metadata is stored
-            metadata_path (str): path to metadata (only folders); the metadata file
-                has the same name as the image (and a .txt ending)
         """
         # cv2 requires BGR format
         self.toBGR()
-        assert self.colorspace == "bgr"
+        assert self.colorspace == "BGR"
 
         # Write image, using the conventional matrix indexing
         cv2.imwrite(str(Path(path + name + file_format)), self.img)
         print("Image saved as: " + str(Path(path + name + file_format)))
 
-        # Write meta data
-        if save_metadata:
-            f = open(str(Path(metadata_path + name + ".txt")), "w")
-            f.write("Origo: " + str(self.origo) + "\n")
-            f.write("Width: " + str(self.width) + "\n")
-            f.write("Height: " + str(self.height) + "\n")
-            f.write("Dimension: " + str(self.dim) + "\n")
-            f.close()
```

Comment:
Agree that the management of `metadata` is not anymore business of `daria.Image`.